include/openroad/OpenRoad.hh

```diff
@@ -98,6 +98,11 @@ namespace pdnsim {
 class PDNSim;
 }
 
+namespace antennachecker {
```

Comment:
I would prefer antenna_checker in general. IE, instead of smashing words together.

Reply:
This was addressed in the new commit.

Reply:
@msaligane you can click on the "resolve conversation" button to mark that comment has been resolved.